airflow/providers/asana/hooks/asana.py

```diff
@@ -27,6 +28,34 @@
 from airflow.hooks.base import BaseHook
 
 
```

Comment:
I want to say that this is the second time I see you using this and it should maybe live in airflow.hooks.base as a shared helper method.  Is there a reason not to do it that way?